tests/test_yield.py

```diff
@@ -0,0 +1,14 @@
+
+def test_error_during_setup(pytester):
```

Comment:
Maybe you meant something like `test_yield` here.

```suggestion
def test_yield(pytester):
```

Reply:
yep